README.md

```diff
@@ -37,19 +60,19 @@ iex> Multihash.encode(0x20, :crypto.hash(:sha, "Hello"))
 
 It's possible to [truncate a digest](https://github.com/jbenet/multihash/issues/1#issuecomment-91783612) by passing an optional `length` parameter. Passing a `length` longer than the default digest length of the hash function will return an error.
 
-### Examples
+#### Examples
 ```
 iex> Multihash.encode(:sha1, :crypto.hash(:sha, "Hello"), 10)
 {:ok, <<17, 10, 247, 255, 158, 139, 123, 178, 224, 155, 112, 147>>}
 iex> Multihash.encode(:sha1, :crypto.hash(:sha, "Hello"), 30)
 {:error, "Invalid digest length"}
 ```
 
-## Decoding
+### Decoding
 
 Decode the provided multihash to %Multihash{code: , name: , length: , digest: }
```

Comment:
Lets put it under the ` code markdown tag with the type information

`%Multihash{name: atom, code: integer, length: integer, digest: bitstring}`